src/Microsoft.AspNet.Mvc.Razor/HelperResult.cs

```diff
@@ -3,30 +3,33 @@
 
 using System;
 using System.IO;
+using Microsoft.AspNet.Html.Abstractions;
 using Microsoft.Framework.Internal;
+using Microsoft.Framework.WebEncoders;
 
 namespace Microsoft.AspNet.Mvc.Razor
 {
     /// <summary>
     /// Represents a deferred write operation in a <see cref="RazorPage"/>.
     /// </summary>
-    public class HelperResult
+    public class HelperResult : IHtmlContent
     {
-        private readonly Action<TextWriter> _action;
+        private readonly Action<TextWriter, IHtmlEncoder> _action;
```

Comment:
Would it be simpler to just leave this as `Func<TextWriter>`? We're planning to delete this class anyway. How big of a change would that be to include in this PR?